listenbrainz/webserver/templates/index/api-docs.html

```diff
@@ -12,4 +12,10 @@ <h2>Client Libraries</h2>
     <li><em>Haskell</em>: <a href="http://hackage.haskell.org/package/listenbrainz-client"><code>listenbrainz-client</code></a></li>
   </ul>
 
+  <h2>Listen Submitters</h2>
+    These are programs that submit listens to ListenBrainz.
+  <ul>
+    <li><em>mpd</em>: <a href="https://github.com/kori/libra"><code>libra</code></a></li>
```

Comment:
Might be good to add a link to mpd here so that readers and learn about it.

Reply:
Alright, I did that in commit 65e1c2a5de99b16b36f8be419ccde07a8c514b08